patterns/disposer.md

```diff
@@ -0,0 +1,84 @@
+# The Promise Disposer Pattern
```

Comment:
To be fair, this looks very fishy. Googling "promise disposer" returns a single meaningful result, which is your question on stackoverflow. In this question you claim that you "can't figure out what it is" , but than immediately answer the question yourself, providing link to those issues, posted almost a year before.

Maybe there is a better known name for this pattern?